src/Components/AddressForm.tsx

```diff
@@ -120,19 +120,22 @@ export class AddressForm extends React.Component<
             </Serif>
             <CountrySelect
               selected={
-                lockCountryToOrigin || lockCountriesToEU
+                lockCountryToOrigin ||
+                (lockCountriesToEU && !this.state.address.country)
```

Comment:
#minor Would this be cleaner if we pulled it out into another `const` above? Something like `useCountryFromState` vs `useCountryFromProps`?